x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -656,6 +656,20 @@ public void loadExtensions(ExtensionLoader loader) {
         Property.NodeScope
     );
 
+    // Serverless only - trigger scale up if necessary
+    public static final Setting<ByteSizeValue> DUMMY_ENTITY_MEMORY = Setting.memorySizeSetting(
+        "xpack.ml.dummy_entity_memory",
+        ByteSizeValue.ofBytes(0),
+        Setting.Property.Dynamic,
+        Setting.Property.NodeScope
+    );
+    public static final Setting<Integer> DUMMY_ENTITY_PROCESSORS = Setting.intSetting(
+        "xpack.ml.dummy_entity_processors",
+        0,
```

Comment:
It's worth adding a minimum value to prevent negatives.

```suggestion
        0, 0
```